reserver/reserver_param.py

```diff
@@ -1,7 +1,24 @@
 # -*- coding: utf-8 -*-
 """Parameters and constants."""
+OVERVIEW = """
+A Python that offers the ability to quickly reserve a PyPI package name.
```

Comment:
Wrong overview.

`Reserver is an open source Python package that offers the ability to quickly reserve a PyPI package name. Got a notion? Before it's taken, immediately reserve the product name!`

Reply:
Or

`A Python package that offers the ability to quickly reserve a PyPI package name. Got a notion? Before it's taken, immediately reserve the product name!`